benchexec/pytest_analyze_run_result.py

```diff
@@ -0,0 +1,197 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import sys
+import pytest
+import types
+
+from benchexec.util import ProcessExitCode
+from benchexec.model import Run
+from benchexec.result import (
+    RESULT_FALSE_REACH,
+    RESULT_ERROR,
+    RESULT_UNKNOWN,
+    RESULT_TRUE_PROP,
+)
+from benchexec.tools.template import BaseTool
+
+sys.dont_write_bytecode = True  # prevent creation of .pyc files
```

Comment:
I don't think we need this in test code. It was intended to prevent Python writing all the `.pyc` files out for regular users.